### PR TITLE
new lab inheritance and constructors

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,30 +2,166 @@
...
+
+## Task #2 - Implement the `Cat` subclass
+
+Create a new class named `Cat` that extends `Pet`.
+
+- `Cat` should have an additional instance variable `purring` of type `boolean`.
+- We will assume all cats **do not** like baths, so the `Cat` constructor only takes
+  2 parameters: `name` and `purring`.
+    - Call the superclass constructor to initialize the inherited `name` and `likesBaths` fields.
+      Remember, cats do not like baths!
```
Comment (kcleland0818) [inline README.md:112]: Fun fact about my cat, Riley! She loves water! She takes a shower with me almost every day and constantly asks the water to be turned on in the sink so she can play with it haha! 

I recognize her as the outlier of cats, as my other cat is normal and does not like baths - but does dramatically tolerate them.

**Review by kcleland0818 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (linda-seiter) [reply README.md:112]: My son's cat actually loves water too and likes to sit in the kitchen sink!

Comment (kcleland0818) [reply README.md:112]: Extension of the inheritance lab - Create a class called WaterCat for the unusual cats that like water. Haha just kidding!

Comment (linda-seiter) [reply README.md:112]: love it!

**Review by developerjaym — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**